dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/DescribeStep.java

```diff
@@ -136,6 +150,17 @@ public void doPatchProcessing(Context context, HttpServletRequest currentRequest
             String[] split = patchOperation.getAbsolutePath(op.getPath()).split("/");
             if (inputConfig.isFieldPresent(split[0])) {
                 patchOperation.perform(context, currentRequest, source, op);
+                String mappedToIfNotDefault = this.getMappedToIfNotDefault(split[0], inputConfig);
+                // if the complex input field contains `mapped-to-if-not-default` definition
+                // put additional data to the defined metadata from the `mapped-to-if-not-default`
+                if (!StringUtils.isBlank(mappedToIfNotDefault)) {
```

Comment:
throw exception if the condition is met, otherwise continue with code execution. make it more readable

Reply:
fixed